### PR TITLE
Selectively revert suspect commits causing API instability

### PR DESCRIPTION
**Pull request by sarayourfriend**
## Description

Selectively reverts commits that we suspect are introducing instability into the most recent version of the API.

Notably, the logging is only reverted in the search controller area to facilitating applying `git revert` against the search controller refactor. The logging changes are kept in the other parts of the application. Likewise, the fix from https://github.com/WordPress/openverse-api/pull/730 is also preserved by moving it into the `search_controller.py`, applied during the revert of the search controller refactor.

It also applies the Elasticsearch configuration changes from https://github.com/WordPress/openverse-api/pull/712 to the old `search_controller.py` module. This preserves that change in other parts of the app and moves it forward into the old search controller.

## Testing Instructions

CI, smoke test locally.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [N/A] I added or updated tests for the changes I made (if applicable).
- [N/A] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: api/test/audio_integration_test.py [hunk 1]
```diff
@@ -125,5 +125,5 @@ def test_audio_uuid_validation():
     uuid_validation("audio", "abcd")
 
 
-def test_image_related(audio_fixture):
+def test_audio_related(audio_fixture):
```
Comment (sarayourfriend) [inline api/test/audio_integration_test.py:128]: Just fixing a typo :sweat_smile:

**Review by AetherUnbound — APPROVED** [review]
The reverts look good. I went through and checked the partial revert and verified that the changes were restricted to the ES module(s). Tests & basic interaction ran correctly locally.

File: api/catalog/api/serializers/media_serializers.py [hunk 2]
```diff
@@ -107,22 +106,16 @@ class MediaSearchRequestSerializer(serializers.Serializer):
...
-        help_text="The index of the page of the results to show.",
-    )
     page_size = serializers.IntegerField(
-        min_value=1,
-        max_value=500,
-        default=20,
-        help_text="The number of results to show in one page.",
+        label="page_size",
+        help_text="Number of results to return per page.",
+        required=False,
```
Comment (krysal) [inline api/catalog/api/serializers/media_serializers.py:112]: We should keep this validation here as is the most appropriate place, though now the validation on the `api/catalog/api/utils/pagination.py` file was re-added so it's not blocking this.

**Review by krysal — APPROVED** [review]
Looks good to me. Did some requests manually and everything is working as expected.

**Review by stacimc — COMMENTED** [review]
A few minutes too slow to approve, but this also looked good to me with local tests and basic API requests.

**Pull request merged**